localenv/mock-account-servicing-entity/app/entry.server.tsx

```diff
@@ -30,6 +30,15 @@ async function callWithRetry(fn: () => any, depth = 0): Promise<void> {
 }
 
 if (!global.__seeded) {
+  const tenantId = process.env.OPERATOR_TENANT_ID
+  const apiSecret = process.env.SIGNATURE_SECRET
```

Comment:
It doesn't seem like `SIGNATURE_SECRET` is set in the environment anymore.

Reply:
It was removed from the frontend - is that what you're referring to? The docker compose sets them for the mock ase. When I spin up rafiki it doesnt error on the lines below for the secret, and it spits out a link with the `apiSecret`.

Reply:
Ah, got it. My mistake.